### PR TITLE
Improve export script

### PR DESCRIPTION
**Pull request by davidgasquez**
Hey there! 

Made a bunch of improvements to the `export.sh` script and added them all in this PR. 

It supports both `sentinel-archiver` and `lily job run walk` jobs.

### CODE REVIEW

File: scripts/export.sh [hunk 1]
```diff
@@ -30,19 +31,34 @@ nohup lily daemon --repo="${REPO_PATH}" --config=/lily/config.toml --bootstrap=f
...
+# Add WALKEPOCHS to the FROM_EPOCH
 TO_EPOCH=$((FROM_EPOCH + WALK_EPOCHS))
+# TO_EPOCH=$(echo "${STATE}" | jq -r ".summary.stateroots.newest")
 
 echo "Walking from epoch ${FROM_EPOCH} to ${TO_EPOCH}"
 sleep 10
 
 # Run export
-# archiver run --storage-path /tmp/data --ship-path "${EXPORT_DIR}" --min-height="${FROM_EPOCH}" --max-height="${TO_EPOCH}"
+# archiver export --storage-path /tmp/data --ship-path "${EXPORT_DIR}" --min-height="${FROM_EPOCH}" --max-height="${TO_EPOCH}"
```
Comment (davidgasquez) [inline scripts/export.sh]: Once we validate that the `export` command is working properly, we can basically comment out/delete the rest of the script!

File: scripts/export.sh [hunk 2]
```diff
@@ -30,34 +31,55 @@ nohup lily daemon --repo="${REPO_PATH}" --config=/lily/config.toml --bootstrap=f
 # Wait for Lily to come online
 lily wait-api
 
-# Extract the walking epochs
-FROM_EPOCH=$(echo "${SNAPSHOT_FILE}" | cut -d'_' -f2)
+# Extract the available walking epochs
+STATE=$(lily chain state-inspect -l 3000)
+# FROM_EPOCH=$(echo "${SNAPSHOT_FILE}" | cut -d'_' -f2)
+FROM_EPOCH=$(echo "${STATE}" | jq -r ".summary.stateroots.oldest")
+FROM_EPOCH=$((FROM_EPOCH + 1))
```
Comment (davidgasquez) [inline scripts/export.sh:38]: Walking with a one epoch shift to avoid errors on the first epoch of each export.

File: Dockerfile [hunk 3]
```diff
@@ -12,7 +12,7 @@ RUN apt-get update -y && \
 
 WORKDIR $SRC_PATH
 
-RUN git clone https://github.com/filecoin-project/sentinel-archiver.git && \
+RUN git clone --branch kasteph/11 https://github.com/filecoin-project/sentinel-archiver.git && \
```
Comment (kasteph) [inline Dockerfile:15]: This branch has been merged :D

File: Dockerfile [hunk 4]
```diff
@@ -22,7 +22,7 @@ FROM buildpack-deps:buster-curl
 
 # Install aria2 and zstd
```
Comment (kasteph) [inline Dockerfile:23]:
```suggestion
```
Maybe L25 is good enough after all

**Review by kasteph — APPROVED** [review]

**Pull request merged**